docs/technical-docs/actions.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Python CI Workflow
+parent: Technical Documentation
+---
+
+# Python CI Workflow
+
+## Overview
+
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) is a tool available to
+GitHub projects and is used to perform Continuous Integration (CI) tasks. Tasks like
+[linting](https://en.wikipedia.org/wiki/Lint_(software)) code and checking if the code complies with
+lab styling standards.
+
+For python, we will use [pylint](https://pylint.org/) which will test code for syntactic errors and
+[pycodestyle](https://github.com/pycqa/pycodestyle) for the style guide checks. These are the two
+checks that will be used on all projects that contain Python code.
+
+## Using GitHub Actions on a Repository
+
+To start a new Python project, create a new project and copy or download this
+[template](#python-ci-workflow). This will include a [YAML](https://en.wikipedia.org/wiki/YAML)
+configuration file, a shell script, and a [pip](https://pip.pypa.io/en/stable/) dependency text
+file.
+
+The YAML configuration file will have the options to run the GitHub Action. The shell script
+`run_tests.sh` will be a Bash script that is called from the GitHub Action virtual machine to run
+the Python checks. The `requirements-dev.txt` file is a
+[requirements](https://pip.pypa.io/en/stable/reference/pip_install/#example-requirements-file) file
+to install the Python dependencies needed using `pip` to run the checks.
+
+## Using GitHub Actions
+
+Once you upload changes you can access the GitHub Actions by going to the "Actions" tab:
+
+![actions](https://raw.githubusercontent.com/NDCLab/wiki/gh-pages/docs/_assets/gh_actions/actions.png)
+
+You will have workflows on the left, this are custom named CI workflows. By default all Python
```

Comment:
this should be these?

Is there a way for me to make/propose changes directly?